docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -15,21 +15,32 @@ endif::[]
 
 // tag::how-remote-clusters-work[]
```

Comment:
The content is this tagged region gets re-used here:
https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-cross-cluster-search.html#ccs-works

Originally, that was intended to explain the concept of gateway nodes _before_ the recommendations outlined here:
https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-cross-cluster-search.html#ccs-gateway-seed-nodes

With this changes, I don't think this re-use makes sense anymore. I'm also not sure if those recommendations would work with proxy mode.

Could you advise on how these changes would impact those CCS recommendations? After that, you can either update those docs as a part of this PR or I can update them as part of a separate PR. Thanks!